tests/controllers/user.test.js

```diff
@@ -30,6 +29,71 @@ function expectResponseRenderedSubpagePartial(resp, partial) {
 }
 
 
+test("user getDashboard with valid user renders dashboard with verified emails and their breaches", async () => {
+  const testSubscriberEmail = TEST_SUBSCRIBERS.firefox_account.primary_email;
+  const testSubscriber = await DB.getSubscriberByEmail(testSubscriberEmail);
+  const mockReturnedBreaches = testBreaches.slice(0,2);
+  HIBP.getBreachesForEmail = jest.fn();
+  HIBP.getBreachesForEmail.mockReturnValue(mockReturnedBreaches);
+
+  const req = {
+    app: { locals: { breaches: testBreaches } },
+    csrfToken: jest.fn(),
+    fluentFormat: jest.fn(),
+    session: { user: testSubscriber },
+    user: testSubscriber,
+  };
+  const resp = httpMocks.createResponse();
+  resp.render = jest.fn();
+
+  await user.getDashboard(req, resp);
+
+  expect(resp.statusCode).toEqual(200);
+  expect(resp.render).toHaveBeenCalledTimes(1);
+  const renderCallArgs = resp.render.mock.calls[0];
+  expect(renderCallArgs[0]).toEqual("dashboards");
+  const renderCallContext = renderCallArgs[1];
+  expect(renderCallContext.whichPartial).toEqual("dashboards/breaches-dash");
+
+  expect(renderCallContext.verifiedEmails.length).toEqual(2);
+  expect(renderCallContext.unverifiedEmails.length).toEqual(1);
+  expect(renderCallContext.verifiedEmails[0].breaches.length).toEqual(mockReturnedBreaches.length);
+});
+
+
+test("user getDashboard with valid user renders verified emails and resolved breaches", async () => {
+  const testSubscriberEmail = TEST_SUBSCRIBERS.firefox_account.primary_email;
+  const testSubscriber = await DB.getSubscriberByEmail(testSubscriberEmail);
+  // TODO: figure out how the F to put resolved_breaches in the test db seed?
```

Comment:
When I try to add a `resolved_breaches: {"firefoxaccount@test.com": [0]},` line in the `firefox_account` object of `TEST_SUBSCRIBERS` in the `db/seeds/test_subscribers.js` file, knex can't seem to insert into the `subscribers` table. 😢